source/02-FN/03.ptx

```diff
@@ -11,6 +11,657 @@
+                  <li> <m>(0, -3)</m> </li>
+              <li> <m>(-1, 0)</m> </li>
+              <li> <m>(3, 0)</m> </li>
+              <li> <m>(-3, 0)</m> </li></ol></p>
+              <answer>
+                <p>
+                  Choice A: <m>(0,-3)</m>
+                </p>
+              </answer>
+              </task>
+              <task>
+                  <p>
+                   Suppose you are told that a function has the following intercepts: 
+                   <ul>
+                    <li>
+                        <m>x</m>-intercepts: <m>(-2,0)</m> and <m>(6,0)</m>
+                    </li>
+                    <li>
+                      <m>y</m>-intercept: <m>(0,4)</m>
+                  </li>
+                   </ul> 
+                  </p>
+                  <p>What would the graph look like?
+                  </p>
+                  <answer>
+                    <p>
+                      Answers will vary.
+                    </p>
+                  </answer>
+              </task>
+              <task>
+                  <p>
+                    What would a graph look like if it had more than one <m>y</m>-intercept?
+                  </p>
+                  <answer>
+                    <p>
+                      You could draw a variety of graphs, but they all would not be functions.
+                    </p>
+                  </answer>
+              </task>
+    </activity>
+
+    <remark>
+      Notice in <xref ref="identify-intercepts-from-graph"/>, that a function can have multiple <m>x</m>-intercepts, but only one <m>y</m>-intercept. Having more than one <m>y</m>-intercept would create a graph that is not a function!
+    </remark>
+
+    <definition xml:id="def-domain-and-range-of-graphs">
+      <statement>
+        <p>
+          The <term>domain</term> refers to the set of possible input values and the <term>range</term> refers to the set of possible output values. If given a graph, however, it would be impossible to list out all the values for the domain and range so we use interval notation to represent the set of values.
+        </p>
+      </statement>
+
+      <aside>
+        <p>Recall that the terms <term>domain</term> and <term>range</term> were first introduced in <xref ref="def-relation"/>.</p>
+      </aside>
+    </definition>
+
+    <activity xml:id="find-domain-and-range-from-graph">
+      <introduction>
+        <p>
+          Use the following graph to answer the questions below.
+        </p>
+        <figure xml:id="graph-of-line-segment">
+          <!--<image source="Images/Graph-with-two-endpoints.jpg"/>-->
+          <image>
+            <sageplot>
+              f(x) = 9/8*(x-4)+4
+              p=plot(f, (x, -4, 4), ymin=-7, ymax=7, xmin=-5,xmax=5,color='blue', thickness=3,)
+              p+=point((4,4),pointsize=50,color='blue')
+              p+=point((-4,-5),pointsize=50,color='blue')
+              p
+            </sageplot>
+          </image>
```

Comment:
I think we should use `<sageplot>`s for these for consistency. I went ahead and did this one, but the others need to be done as well.

Reply:
I've added the others.